### PR TITLE
Fix bug 1452647: Add Tags tab to the Localization dashboard

### PR DESCRIPTION
**Pull request by mathjazz**
In https://github.com/mozilla/pontoon/commit/5c4152d2a33be7a0c28d766b8da922a9d1fd35f6 we landed support for the _Tags_ tab in the [project dashboard](https://mozilla-pontoon-staging.herokuapp.com/projects/firefox/tags/), which contains a list of tags and their stats for the project. This patch bring the _Tags_ tab to the [localization dashboard](https://mozilla-pontoon-staging.herokuapp.com/sl/firefox/tags/), which contains a list of tags for the project and their stats for the localization (project-locale).

[First commit](https://github.com/mozilla/pontoon/commit/43fc906227d8442d533b736b7e786553a8c0cf60) sets the stage by generalizing the code for Tags tab on the Project dashboard. The [second one](https://github.com/mozilla/pontoon/commit/4aeaa0b538e7012689bc067ce4c779687b56124a) is an actual bugfix. It will be easier to review them separtely.

Check out the [docs](https://github.com/mozilla-l10n/localizer-documentation/pull/120/) for more information about tags.

### CODE REVIEW

**Comment by mathjazz** [comment]
@flodolo I've deployed this to stage. Could you take it for a spin?
https://mozilla-pontoon-staging.herokuapp.com/sl/firefox/tags/

**Comment by flodolo** [comment]
I'll set aside time to review this on Monday. It's great to see us unblocking tags.

**Comment by flodolo** [comment]
I've been playing with it for a bit, and can't really spot anything wrong 👍

**Comment by mathjazz** [comment]
@jotes r? Please see the [description](https://github.com/mozilla/pontoon/pull/969#issue-189066225) for more details on review and testing.

**Comment by jotes** [comment]
@mathjazz if that's not urgent then I'll be able to review this patch around 7PM-8PM today.

**Comment by mathjazz** [comment]
Excellent, it can wait until then.

File: pontoon/localizations/views.py [hunk 1]
```diff
@@ -108,6 +125,28 @@ def ajax_resources(request, code, slug):
     })
 
 
+@require_AJAX
+def ajax_tags(request, code, slug):
+    """Tags tab."""
+    locale = get_object_or_404(Locale, code=code)
+    project = get_object_or_404(Project, slug=slug)
+
+    if not project.tags_enabled:
```
Comment (jotes) [inline pontoon/localizations/views.py]: [nit] Class-based views could help generalize that logic and move it into one place.

File: pontoon/localizations/views.py [hunk 2]
```diff
@@ -25,6 +35,13 @@ def localization(request, code, slug):
         'project': project,
         'project_locale': project_locale,
         'resource_count': resource_count,
+        'tags': (
+            len(
+                TagsTool(projects=[project], locales=[locale], priority=True)
+            ) or False
```
Comment (jotes) [inline pontoon/localizations/views.py]: `or False` is required?

File: pontoon/localizations/views.py (hunk 1 above)
Comment (mathjazz) [reply pontoon/localizations/views.py]: I'll leave that refactor for later, we might have more related views coming.

**Pull request merged**